bin/emqx

```diff
@@ -38,14 +46,17 @@ export PROGNAME="erl"
 export ERTS_LIB_DIR="$RUNNER_ROOT_DIR/lib"
 DYNLIBS_DIR="$RUNNER_ROOT_DIR/dynlibs"
 
-# Echo to stderr on errors
-echoerr() {
-    echo -e "$*" 1>&2;
+logerr() {
+    echo -e "$(tput setaf 1)ERROR: $*$(tput sgr0)" 1>&2
+}
+
+logwarn() {
+    echo "$(tput setaf 3)WARNING: $*$(tput sgr0)"
```

Comment:
Q: this one doesn't need `-e` and `1>&2`?

Reply:
I'm not sure why we had `-e` in `echoerr` in the first place, just left it as is.
As for the redirection, I think it's ok to have warnings on stdout - same as above, before this change we were printing warnings on stdout and I did not change that.